build/webpack.base.conf.js

```diff
@@ -12,7 +12,7 @@ var gitRevisionPlugin = new GitRevisionPlugin({
 const isProd = process.env.NODE_ENV === 'production';
 const HardSourceWebpackPlugin = require('hard-source-webpack-plugin')
 
-function resolve (dir) {
+function resolve(dir) {
```

Comment:
@eddieferrer @ryan-ludwig All the updates in this page are weird... Our auto-fix settings should be inherited from the global settings to prevent this type of churn.

Reply:
Oh, yeah looks like I formatted the file at some point.

Reply:
Yes please revert changes to this file in this branch for now...